### PR TITLE
Upgrading to v2.5.0

### PR DESCRIPTION
**Pull request by metinavc1**

### CODE REVIEW

File: tablepos/TablePos/Logic/TableApp+SPIPayAtTableDelegate.swift [hunk 1]
```diff
@@ -64,4 +75,51 @@ extension TableApp: SPIPayAtTableDelegate {
         return response
     }
     
+    func pay(atTableGetOpenTables operatorId: String!) -> SPIGetOpenTablesResponse! {
+        let response: SPIGetOpenTablesResponse = SPIGetOpenTablesResponse()
+        let openTablesArray = NSMutableArray()
```
Comment (chrishulbert) [inline tablepos/TablePos/Logic/TableApp+SPIPayAtTableDelegate.swift:78]:
```suggestion
        var openTablesArray: [SPIOpenTablesEntry] = []
```
Couldn't this work? Or does response.openTablesData require an objc-style NSArray?

**Review by chrishulbert — APPROVED** [review]

File: tablepos/TablePos/Logic/TableApp+SPIPayAtTableDelegate.swift (hunk 1 above)
Comment (metinavc1) [reply tablepos/TablePos/Logic/TableApp+SPIPayAtTableDelegate.swift:78]: it couldn't work with that. NSMutableArray is ok

File: motelpos/MotelPos/View Controllers/Main/MainViewController+Actions.swift [hunk 2]
```diff
@@ -103,11 +149,33 @@ extension MainViewController {
         MotelApp.current.client.initiateRecovery(referenceId, transactionType: .getLastTransaction, completion: printResult)
     }
     
+    func sanitizePrintText(_ text: String?) -> String? {
+        let boldText: String? = text?.replacingOccurrences(of: "\\n", with: "\n");
```
Comment (chrishulbert) [inline motelpos/MotelPos/View Controllers/Main/MainViewController+Actions.swift:153]: Could these 3 lines simply be replaced with one call to:
replacingOccurrences(of: "\\", with: "\");

**Review by chrishulbert — APPROVED** [review]

File: motelpos/MotelPos/View Controllers/Main/MainViewController+Actions.swift (hunk 2 above)
Comment (metinavc1) [reply motelpos/MotelPos/View Controllers/Main/MainViewController+Actions.swift:153]: i tried it but i doesn't work like that

**Pull request merged**